ersilia/default.py

```diff
@@ -9,8 +9,10 @@
     os.makedirs(EOS)
 ROOT = os.path.dirname(os.path.realpath(__file__))
 BENTOML_PATH = os.path.join(str(Path.home()), "bentoml")
+PYTHON_VERSION = "3.10" # default
```

Comment:
What's this for?

Reply:
It was used for previous code, when I create conda venv for the models. You are amazing catcher @DhanshreeA.